api/tests/unit/application/certification-centers/index_test.js

```diff
@@ -314,4 +319,45 @@ describe('Unit | Router | certification-center-router', function () {
       sinon.assert.calledOnce(certificationCenterController.findPaginatedSessionSummaries);
     });
   });
+
+  describe('POST /api/certification-centers/{certificationCenterId}/sessions/import', function () {
+    const testFilePath = `${__dirname}/testFile_temp.csv`;
+
+    let headers;
+    let payload;
+
+    beforeEach(async function () {
+      await writeFile(testFilePath, Buffer.alloc(0));
+      const form = new FormData();
+      const knownLength = await stat(testFilePath).size;
```

Comment:
Je suis etonne que le .size fonctionne. Pour etre plus explicite:
```suggestion
      const { size: knownLength } = await stat(testFilePath);
```